webapp/channels/src/components/autosize_textarea.tsx

```diff
@@ -13,203 +14,169 @@ type Props = {
     onWidthChange?: (width: number) => void;
     onInput?: (e: FormEvent<HTMLTextAreaElement>) => void;
     placeholder?: string;
-    forwardedRef?: ((instance: HTMLTextAreaElement | null) => void) | React.MutableRefObject<HTMLTextAreaElement | null> | null;
```

Comment:
forwardedRef is no longer needed, since we are directly the ref from `forwardRef`.